sdg/outputs/OutputGeoJson.py

```diff
@@ -0,0 +1,263 @@
+import os
+import json
+import copy
+from urllib.request import urlopen
+import sdg
+import pandas as pd
+from sdg.outputs import OutputBase
+
+class OutputGeoJson(OutputBase):
+    """Output SDG data/metadata in GeoJson disaggregated by region."""
+
+
+    def __init__(self, inputs, schema, output_folder='_site', translations=None,
+        geojson_file='regions.geojson', name_property='name', id_property='id',
+        id_column='GeoCode', output_subfolder='regions', filename_prefix='indicator_',
+        exclude_columns=None, id_replacements=None):
```

Comment:
1. Agreed, I've made a change to that effect.
2. I agree for languages like Javascript that don't support named parameters. But Python does support "keyword arguments" which means that the order of those arguments doesn't matter, and the function calls are self-documenting because they're named. I think that as long as we're taking advantage of Python's keyword arguments, using an argument object would be redundant.